src/GraphQLParser/LexerContext.cs

```diff
@@ -181,16 +179,24 @@ private Token ReadComment()
             );
         }
 
+        // TODO: this method can still be optimized no not allocate at all if block string:
```

Comment:
@Shane32 I am 100% sure that this can be done, but I was not so brave to do it myself :).